Library/Homebrew/requirements/tuntap_requirement.rb

```diff
@@ -7,8 +7,9 @@ class TuntapRequirement < Requirement
   satisfy(:build_env => false) { self.class.binary_tuntap_installed? || Formula["tuntap"].installed? }
 
   def self.binary_tuntap_installed?
-    File.exist?("/Library/Extensions/tun.kext") && File.exist?("/Library/Extensions/tap.kext")
-    File.exist?("/Library/LaunchDaemons/net.sf.tuntaposx.tun.plist")
-    File.exist?("/Library/LaunchDaemons/net.sf.tuntaposx.tap.plist")
+    File.exist?("/Library/Extensions/tun.kext") &&
+      File.exist?("/Library/Extensions/tap.kext") &&
+      File.exist?("/Library/LaunchDaemons/net.sf.tuntaposx.tun.plist") &&
+      File.exist?("/Library/LaunchDaemons/net.sf.tuntaposx.tap.plist")
```

Comment:
Maybe even better as follows? (It feels much more readable to me, despite the two extra lines.)

```ruby
%w[
  /Library/Extensions/tun.kext
  /Library/Extensions/tap.kext
  /Library/LaunchDaemons/net.sf.tuntaposx.tun.plist
  /Library/LaunchDaemons/net.sf.tuntaposx.tap.plist
].all? { |file| File.exist?(file) }
```